code/game/objects/items/storage/firstaid.dm

```diff
@@ -167,6 +167,10 @@
 				usr.s_active.close(usr)
 			src.show_to(usr)
 
+/obj/item/storage/pill_bottle/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is trying to get the cap off [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")

```

Comment:
1. Remove cap from pill bottle
2. Die
3. ???
4. ~~Profit!~~ Still death

This one is kinda lame; it'd also logically mean anytime you open a pill bottle you should die.

Reply:
It's a reference to a movie that I thought most people who browsed /tg/ and the rest of 9gag had seen.

Reply:
>In the end, Taylor, Samantha, Cheyenne, Caleb, and Scott all stand up against Lindsay. Other classmates witnessing Taylor standing up to Lindsay realize how bullying is terrible, and antagonize Lindsay for her abuse. A law is then passed against cyberbullying.

Was this really made

Reply:
Holy shit all it's missing is a "And then everyone in the classroom clapped."